lib/features/postings/postings_firestore.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:instreal/features/postings/posting_firestore_model.dart';
+
+abstract interface class PostingsFirestore {
```

Comment:
Is this the service for communicating with firebase? Shouldn't b generalized for all features not only posting?

Reply:
Yes. this is supposed to be generalized for different stores like firebase, localdb, or other remote stores but for now, the only purpose is for firebase so I didn't make it generalized yet. When we introduce other stores, we can consider refactoring it.

Reply:
Ok, but I think was still better to separate this from the post.